guardian/shortcuts.py

```diff
@@ -159,7 +170,7 @@ def get_perms_for_model(cls):
     return Permission.objects.filter(content_type=ctype)
 
 def get_users_with_perms(obj, attach_perms=False, with_superusers=False,
-        with_group_users=True):
+        with_group_users=True, direct_perms_only=False):
```

Comment:
Personally, I prefer to keep all stuff/discussions in one place (GitHub issues/pull requests). Moreover, I observed that younger developers have issues using mailing lists. They are simply old school.

So I do not see much benefit of having a mailing list.

Maybe we just need label "discussion" for an issue and then we discuss something there.